compiler/damlc/tests/daml-test-files/ChoiceObservers.daml

```diff
@@ -0,0 +1,40 @@
+-- Copyright (c) 2020, Digital Asset (Switzerland) GmbH and/or its affiliates.
+-- All rights reserved.
+
+module ChoiceObservers where
+
+-- @SINCE-LF 1.dev
+
+-- @QUERY-LF [ .modules[] | .templates[] | select(lf::get_template_name($pkg) == ["TheTemplate"]) | .choices[] | select(lf::get_name($pkg) == "C1") | .observers ] == [ null ]
+
+-- @QUERY-LF [ .modules[] | .templates[] | select(lf::get_template_name($pkg) == ["TheTemplate"]) | .choices[] | select(lf::get_name($pkg) == "C2") | .observers ] != [ null ]
```

Comment:
It's probably safer to make a positive statement about the observers field here, something like "it's an object with a field name `foo`". Things can be different for so many reasons.